Demo/PQFCustomLoadersDemo/VBFRotatingSquare.h

```diff
@@ -0,0 +1,33 @@
+//
+//  VBFRotatingSquare.h
+//  VBFFlatLoaders
+//
+//  Created by Victor Baro on 19/09/2014.
+//  Copyright (c) 2014 Victor Baro. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import "PQFLoader.h"
+
```

Comment:
The loaders should be in the folder PQFCustomLoaders in the root (which is the Source for the Pod) instead of inside the one inside the Demo folder